nafas/functions.py

```diff
@@ -221,6 +221,8 @@ def get_input_standard(input_func=input):
             try:
                 input_data[item] = int(input_func(""))
                 exit_flag = True
+            except KeyboardInterrupt:
+                sys.exit()
             except Exception:
                 print("[Error] Bad Input!")
```

Comment:
This is just to make an exception for `KeyboardInterrupt` when the user is inputting, right?
Isn't there any other way for that?
I think we should make it consistent with the other exception in the `main`:

1. Either `print("\n" + EXIT_MESSAGE)` or
2. `rais KeyboardInterrupt `

Inside the `except KeyboardInterrupt`

Reply:
fixed